modules/sdk/project-templates/project-templates-content-targeting-report/src/main/resources/archetype-resources/pom.xml

```diff
@@ -125,4 +126,4 @@
 			</plugin>
 		</plugins>
 	</build>
-</project>
\ No newline at end of file
+</project>
```

Comment:
Did you add an empty line at the end? Please remove it. Also, I think `ProjectTemplateFilesTest` will fail.